### PR TITLE
Add Pickle Finance SDK Adapter

### PR DESCRIPTION
**Pull request by larrythecucumber321**
##### Twitter Link:
https://twitter.com/picklefinance

##### List of audit links if any:
https://github.com/pickle-finance/protocol/tree/master/audits

##### Website Link:
https://app.pickle.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
pickle

##### Current TVL:
$92M

##### Chain:
- Ethereum
- Polygon

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
`pickle-finance`

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
`7022`

##### Description/bio (to be shown on DefiLlama):
The Future of Finance is Green. A cross-chain yield aggregator to help users get the best yield.

##### Token address and ticker if any:
- 0x429881672B9AE42b8EbA0E26cD9C73711b891Ca5
- PICKLE

##### Category (Yield/Dexes/Lending/Minting):
Yield aggregator

##### [Optional] ETH addresss (to receive a LlamaPunk):
0xacfe4511ce883c14c4ea40563f176c3c09b4c47c

### CODE REVIEW

File: projects/helper/unwrapLPs.js [hunk 1]
```diff
@@ -39,6 +86,12 @@ async function unwrapCrv(balances, crvToken, balance3Crv, block, chain = "ethere
         chain,
         abi: 'erc20:balanceOf'
     })).output
+
+    // steth case where balanceOf not applicable on ETH balance
+    if (crvToken === "0x06325440d014e39736583c165c2963ba99faf14e") {
+        underlyingSwapTokens[0].output = underlyingSwapTokens[0].output * 2;
```
Comment (0xngmi) [inline projects/helper/unwrapLPs.js:92]: Isn't `underlyingSwapTokens[0].output` a string?

Comment (0xngmi) [reply projects/helper/unwrapLPs.js:92]: nevermind, I was wrong

**Comment by 0xngmi** [comment]
It seems like alETH's wasn't being unwrapped so I also added some code for that

**Comment by larrythecucumber321** [comment]
Thanks so much @0xngmi 🙏

**Comment by realdealshaman** [comment]
Thanks for your contribution! Your LlamaPunk has been sent! https://opensea.io/0xacfe4511ce883c14c4ea40563f176c3c09b4c47c

**Comment by larrythecucumber321** [comment]
Woohoo thank you for the LlamaPunk!

**Pull request merged**